src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
+        if (userService.findByEmail(email) != null) {
+            User user = userService.findByEmail(email);
```

Comment:
You can simplify your solution. You don't need to call the method `userService.findByEmail` twice 
```suggestion
        User user = userService.findByEmail(email);
        if (user != null) {
```